manifest.json

```
@@ -10,8 +10,8 @@
     "url": "https://nextcloud.com",
     "license": "AGPL-3.0",
     "maintainer": {
-        "name": "YunoHost Contributors",
-        "email": "apps@yunohost.org"
+        "name": "",
+        "email": ""
```

```suggestion
        "name": "Kay0u",
        "email": "pierre@kayou.io"
```
I guess I'm the maintainer now. But it'd be nice if I wasn't the only one :-)